swan-lake/learn-the-platform/ballerina-persist/bal-persist-overview.md

```diff
@@ -5,11 +5,24 @@ description: The section gives an overview of the `bal persist` feature.
 keywords: ballerina, programming language, ballerina packages, persist, data model, cli tool, client api
 permalink: /learn/ballerina-persist/persist-overview/
 active: persist_overview
-intro: The `bal persist` feature allows you to store data in different data stores and retrieve them when needed. A data store can be a database, an in-memory cache, or a file system. The `bal persist` feature currently supports in-memory tables, MySQL databases, and Google Sheets as data stores. The important point is you can use the same syntax to access data in all these data stores. Therefore, you don't need to learn different syntaxes to access data in different data stores.
+intro: The `bal persist` feature allows you to store data in different data stores and retrieve them when needed. A data store can be a database, an in-memory cache, or a file system. The `bal persist` feature currently supports in-memory tables, MySQL and MSSQL databases, and Google Sheets as data stores. The important point is you can use the same syntax to access data in all these data stores. Therefore, you don't need to learn different syntaxes to access data in different data stores.
```

Comment:
We don't support file systems for persist. Should we remove that bit?

Reply:
Fixed in [c6eed56](https://github.com/ballerina-platform/ballerina-dev-website/pull/7147/commits/c6eed56baf0ced5943b8a546ee6055d38484f264)